src/bun.js/test/expect.zig

```diff
@@ -2673,6 +2673,33 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toBeObject(this: *Expect, globalThis: *JSGlobalObject, callFrame: *CallFrame) callconv(.C) JSValue {
+        defer this.postMatch(globalThis);
+
+        const thisValue = callFrame.this();
+        const value: JSValue = this.getValue(globalThis, thisValue, "toBeObject", "") orelse return .zero;
+
+        incrementExpectCallCounter();
+
+        const not = this.flags.not;
+        const pass = value.isObject() != not;
+
+        if (pass) return thisValue;
+
+        var formatter = JSC.ZigConsoleClient.Formatter{ .globalThis = globalThis, .quote_strings = true };
+        const received = value.toFmt(globalThis, &formatter);
+
+        if (not) {
+            const fmt = comptime getSignature("toBeObject", "", true) ++ "\n\n" ++ "Received: <red>{any}<r>\n";
```

Comment:
could you add in a line above `Received: ...` that says `Expected value to be an object\n\n`? Similar to the formatting in `toBeEmpty`